packages/typography/src/Typography.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import Box, { BoxProps } from '@leafygreen-ui/box';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+import { useBaseFontSize } from '@leafygreen-ui/leafygreen-provider';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import omit from 'lodash/omit';
+
+const sharedStyles = css`
+  display: block;
+  margin: unset;
+  font-family: Akzidenz, 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  color: ${uiColors.gray.dark2};
+`;
+
+const h1 = css`
+  font-size: 60px;
+  line-height: 68px;
+  letter-spacing: -0.3px;
+`;
+
+const h2 = css`
+  font-size: 32px;
+  line-height: 40px;
+  letter-spacing: 0px;
+`;
+
+const subtitle = css`
+  font-size: 18px;
+  line-height: 24px;
+  letter-spacing: 0px;
+`;
+
+const typeScale1 = css`
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0px;
+`;
+
+const typeScale2 = css`
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0px;
+`;
+
+const code = css`
+  font-family: 'Source Code Pro', monospace;
+`;
+
+const disclaimer = css`
+  font-size: 12px;
+  line-height: 20px;
+  letter-spacing: 0.2px;
+`;
+
+const overline = css`
+  font-size: 12px;
+  font-weight: 600;
+  text-transform: uppercase;
+  line-height: 16px;
+  letter-spacing: 0.4px;
+`;
+
+type H1Props = HTMLElementProps<'h1'>;
```

Comment:
optional style nit, I'd imagine it's easier to make changes and read code if we had the css, types, and components arranged next to each other, so you can see the full picture all together.

Reply:
+1

Reply:
kk!

Reply:
Done!